src/XMLexport.cpp

```diff
@@ -1127,3 +1127,38 @@ bool XMLexport::writeKey(TKey* pT)
 
     return (isOk && (!hasError()));
 }
+
+bool XMLexport::writeScriptElement(const QString & script)
+{
+    QString localScript = script;
+    localScript.replace(QChar('\x01'), QStringLiteral("\xFFFC\x2401")); // SOH
```

Comment:
Just in case the single Control Picture (or the ORC) code gets used otherwise - the pair together protects against either of the code-points used appearing in isolation. Or do you mean ensuring the Object Replacement Character does not otherwise appear in the data before we try to save?  TBH I have never encountered it in the wild - but I am not everyone...